.github/workflows/pr-labeler.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - uses: actions/checkout@v4 
```

Comment:
This is adding checkout, but not changing labeler.

Reply:
only the config requires changing. v5 of labeler includes the breaking changes, but config was not updates when action was patched

Reply:
Okay, so the checkout step doesn't require any input?

Reply:
no